src/foam/nanos/script/ScriptEvent.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.script',
+  name: 'ScriptEvent',
+
+  documenation: `Captures the running of a Script`,
+
+  implements: [
+    'foam.nanos.auth.LastModifiedByAware'
+  ],
+
+  tableColumns: [
+    'type',
+    'owner',
+    'lastRun',
+    'lastDuration'
+  ],
+
+  properties: [
+    {
+      name: 'id',
+      class: 'String',
+      visibility: 'RO'
+    },
+    {
+      name: 'type',
```

Comment:
Can you document what this field means?

Reply:
And/or, use a more meaningful name.

Reply:
Added. This is similar to the use of 'type' in some other models - useful in tableviews to distinguish subclasses of models shown in the same DAO.

Reply:
renamed to scriptType.